src/main/java/com/expediagroup/dropwizard/resilience4j/Resilience4jBundle.java

```diff
@@ -66,13 +75,17 @@ public Resilience4jBundle(@NonNull Function<T, Resilience4jConfiguration> resili
      * @param resilienceConfiguratorFunction Function to extract the Resilience4j configuration from the dropwizard configuration
      * @param circuitBreakerConfigurator A function that will be passed the name and builder for each circuit breaker before it is created
      * @param retryConfigurator A function that will be passed the name and builder for each retryer
+     * @param timeLimiterConfigurator A function that will be passed the name and builder for each time limiter
      */
     public Resilience4jBundle(@NonNull Function<T, Resilience4jConfiguration> resilienceConfiguratorFunction,
                               @NonNull BiConsumer<String, CircuitBreakerConfig.Builder> circuitBreakerConfigurator,
-                              @NonNull BiConsumer<String, RetryConfig.Builder> retryConfigurator) {
+                              @NonNull BiConsumer<String, RetryConfig.Builder> retryConfigurator,
+                              @NonNull BiConsumer<String, TimeLimiterConfig.Builder> timeLimiterConfigurator
+        ) {
```

Comment:
This constructor is starting to get crazy. Maybe to save as an issue for another PR, but can we simplify this? Does the builder pattern work here?

Reply:
Honestly, having a configurator (`BiConsumer`) for the time limiter seems overkill. I was mostly trying to match the other two.

The circuit breaker and retryer builders have methods for setting predicates for certain things, while the time limiter has no such "advanced" methods --so technically everything can be configured in the config YAML.

So, I can remove the time limiter configurator logic if you agree.

Reply:
Makes sense. Do it!